helpers/StorycapCompare.js

```diff
@@ -0,0 +1,118 @@
+// General imports
+const fs = require('fs');
+const path = require('path');
+const { stdout } = require('process');
+
+// Directory paths
+const APPROVED_DIR_NAME = '.storycap-approved';
+const PENDING_DIR_NAME = '.storycap-pending';
+
+const APPROVED_STORYCAPS = path.join(__dirname, `../${APPROVED_DIR_NAME}`);
+const PENDING_STORYCAPS = path.join(__dirname, `../${PENDING_DIR_NAME}`);
+
+/**
+ * Helper method to get all the file paths from the selected directory recursively.
+ *
+ * @param {string} directory Path to the directory from which to read the files
+ * @returns Array of file paths in the selected directory
+ */
+const getFilesRecursively = (directory) => {
+  const files = [];
+
+  const filesInDirectory = fs.readdirSync(directory);
+
+  filesInDirectory.forEach((file) => {
+    const absolute = path.join(directory, file);
+
+    if (fs.lstatSync(absolute).isDirectory()) {
+      files.push(...getFilesRecursively(absolute));
+    } else {
+      files.push(absolute);
+    }
+  });
+
+  return files;
+};
+
+/**
+ * Helper method to print the information about starting the task and its completion
+ * to the console.
+ *
+ * @param {string} message Message to output before starting the process.
+ * @param {Function} taskToRun Function to invoke.
+ * @returns Value that the function returns
+ */
+const logProcess = (message, taskToRun) => {
+  try {
+    stdout.write(message);
+    const valueToReturn = taskToRun();
+    stdout.write(' DONE ✔\n');
+
+    return valueToReturn;
+  } catch (error) {
+    stdout.write(' ERROR X\n');
+
+    throw error;
+  }
+};
+
+/**
+ * Script's starting point.
+ */
+try {
+  // Get all the approved files.
+  const approvedFiles = logProcess('Fetching approved files...', () => getFilesRecursively(APPROVED_STORYCAPS));
+
+  // Get all the pending files.
+  const pendingFiles = logProcess('Fetching pending files...', () => getFilesRecursively(PENDING_STORYCAPS));
+
+  console.log();
```

Comment:
Why?

Reply:
I think that doing the empty lines this way is more declarative than appending or prepending the `\n` on the surrounding `console.log()` statements.